### PR TITLE
+ ruby31.y: branch parser

### PR DESCRIPTION
**Pull request by koic**
ruby31.y is based on ruby30.y as Ruby 3.0.1. And Ruby 3.0.0 and 3.0.1 are the same (#791) .

### CODE REVIEW

File: lib/parser/ruby31.y [hunk 1]
```diff
@@ -0,0 +1,3077 @@
...
+end
+
+---- header
+
+require 'parser'
+
+---- inner
+
+  def version
+    30
```
Comment (iliabylich) [inline lib/parser/ruby31.y:3066]: This line should be updated too

File: lib/parser/current.rb [hunk 2]
```diff
@@ -92,6 +92,15 @@ def warn_syntax_deviation(feature, version)
     require 'parser/ruby30'
     CurrentRuby = Ruby30
 
+  when /^3\.1\./
+    current_version = '3.1.0'
```
Comment (iliabylich) [inline lib/parser/current.rb:96]: Also, could you change it to `3.1.0-dev` please?

**Comment by koic** [comment]
@iliabylich Thanks for your review! I updated this PR.

**Comment by iliabylich** [comment]
Thanks!

**Pull request merged**